spec/immutable-data-structures.html

```diff
@@ -256,7 +256,7 @@ <h1>Tuple.prototype.concat ( ..._args_ )</h1>
           1. Let _items_ be a List whose first element is _T_ and whose subsequent element are, in left to right order, the arguments that were passed to this function invocation.
           1. Repeat, while _items_ is not empty,
             1. Remove the first element from _items_ and let _E_ be the value of the element.
-            1. Let _spreadable_ be ? IsConcatSpreadable(_E_).
+            1. Let _spreadable_ be ! IsArrayOrTuple(_E_).
```

Comment:
```suggestion
            1. Let _spreadable_ be ? IsArrayOrTuple(_E_).
```

This can throw when calling `#[].concat(p)`, where `p` is a revoked proxy.